frontend/src/components/PopOverComponent.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Box, Typography, Button, Popover } from '@mui/material';
+
+const PopoverComponent = ({ open, handleClose, selectedPlaceProperties, anchorElRef }) => {
+  return (
```

Comment:
Given your use of material-ui, it's preferable to employ controlled components like Box, rather than relying solely on the vanilla div element